remarks/remarks.py

```diff
@@ -39,153 +41,160 @@ def run_remarks(
     combined_pdf=False,
     modified_pdf=False,
 ):
-    for path in pathlib.Path(f"{input_dir}/").glob("*.pdf"):
-        pages = list_pages_uuids(path)
-        name = get_visible_name(path)
-        rm_files = list_ann_rm_files(path)
+    for path in pathlib.Path(f"{input_dir}/").glob("*.metadata"):
+        if not is_document(path):
+            return
```

Comment:
@clement-elbaz, we must have `continue` here, not `return`. A `return` stops the loop through the `xochitl` directory too early. We do not want that as we would miss some files. Can you please fix that in a new commit? I then will merge it. Thanks!

Reply:
Woops! Sorry for that, and thank you for catching it! I meant to type `continue` there but I slipped. Testing the codebase on the `demo` directory did not raise problems so I went along.

I've now force-pushed the correction, the PR should be mergeable now.

This raises two thoughts for me:

1. I initially thought that maintaining a single commit for the whole duration of the PR and force-pushing it with modifications related to your comments would be the simplest approach. But after two corrections I'm not so convinced now: in the future I will probably add my corrections as subsequent commits, then either I or you can rebase the commits as a single pretty commit just before merging. Let me know if you see a problem with that.

2. No promises here, but would you be opened to PRs related to building an integration test suites? Such basic regressions could be caught pretty easily with basic tests.

Thanks!

Reply:
Re (1), either way is fine. Rebasing wouldn't be a problem, but you force-pushing is very convenient :)

Re (2), you are very correct. We absolutely need a suite of tests. I will open a new issue referencing this. If not you or me, maybe someone could jump in and help out sooner than later.